src/app/(frontend)/contact/page.tsx

```diff
@@ -0,0 +1,65 @@
+'use client'
+
+import PageClient from "./page.client"
+import React, { useState, useEffect } from "react"
+import { FormBlock } from "@/blocks/Form/Component"
+import type { Form as FormType } from "@payloadcms/plugin-form-builder/types"
+
+const ContactForm = () => {
+  const [formData, setFormData] = useState<FormType | null>(null)
+  const [error, setError] = useState<string | null>(null)
+  const formTitle = "Contact"
+
+  useEffect(() => {
+    const fetchFormData = async () => {
+      try {
+        const response = await fetch(`/api/forms?where[title][equals]=${encodeURIComponent(formTitle)}`)
+        if (!response.ok) {
+          throw new Error('Failed to fetch form data')
+        }
+        const data = await response.json()
+        if (data.docs && data.docs.length > 0) {
+          setFormData(data.docs[0])
+        } else {
+          setFormData(null)
+        }
```

Comment:
This block is good JS. For TS like this project, I usually do this
```suggestion
const doc = (data?.docs || []).length > 0 ? data.docs[0] : null;
setFormData(doc);
```

Reply:
That sounds interesting! I'll add this in my next commit. I have a question though: Why do you implement this in TS but not in JS? What's the advantage?

Reply:
The file is tsx 😄 and the project use TS, not JS. You can google or ask your LLM why we should use TS